pkg/backends/ibmsecretsmanager.go

```diff
@@ -536,18 +541,64 @@ func storeSecret(secrets *map[string]interface{}, result map[string]interface{})
 	return nil
 }
 
+func (i *IBMSecretsManager) ResolveGroup(group string) (string, error) {
```

Comment:
Does this function need to be exported outside the package? If not, let's keep it private like the others here

Reply:
fixed.